bundles/org.palladiosimulator.textual.tpcm/src/org/palladiosimulator/textual/tpcm/GenerateTPCM.mwe2

```diff
@@ -0,0 +1,93 @@
+module org.palladiosimulator.textual.tpcm.GenerateTPCM
+
+import org.eclipse.xtext.xtext.generator.*
+import org.eclipse.xtext.xtext.generator.model.project.*
+
+import tools.mdsd.ecoreworkflow.mwe2lib.xtext.config.TychoStandardLayoutProjectConfig
+
+var rootPath = "../.."
+
+Workflow {
+    bean=org.eclipse.emf.mwe.utils.StandaloneSetup {
+        scanClassPath=true
+        platformUri=rootPath
+        uriMap = {
+            from = "platform:/plugin/org.eclipse.uml2.codegen.ecore/model/GenModel.ecore"
+            to = "platform:/resource/org.eclipse.uml2.codegen.ecore/model/GenModel.ecore"
+        }
+        uriMap = {
+            from = "platform:/plugin/de.uka.ipd.sdq.stoex/model/stoex.ecore"
+            to = "platform:/resource/de.uka.ipd.sdq.stoex/model/stoex.ecore"
+        }
+        uriMap = {
+            from = "platform:/plugin/de.uka.ipd.sdq.probfunction/model/ProbabilityFunction.ecore"
+            to = "platform:/resource/de.uka.ipd.sdq.probfunction/model/ProbabilityFunction.ecore"
+        }
+        uriMap = {
+            from = "platform:/plugin/de.uka.ipd.sdq.units/model/Units.ecore"
+            to = "platform:/resource/de.uka.ipd.sdq.units/model/Units.ecore"
+        }
+        registerGeneratedEPackage = "de.uka.ipd.sdq.stoex.StoexPackage"
+        registerGeneratedEPackage = "de.uka.ipd.sdq.probfunction.ProbfunctionPackage"
+        registerGeneratedEPackage = "de.uka.ipd.sdq.units.UnitsPackage"
+        registerGeneratedEPackage = "org.eclipse.uml2.codegen.ecore.genmodel.GenModelPackage"
+        registerGenModelFile = "platform:/resource/de.uka.ipd.sdq.stoex/model/stoex.genmodel"
+        registerGenModelFile = "platform:/resource/de.uka.ipd.sdq.probfunction/model/ProbabilityFunction.genmodel"
+        registerGenModelFile = "platform:/resource/de.uka.ipd.sdq.units/model/Units.genmodel"
+        registerGenModelFile = "platform:/resource/org.eclipse.uml2.codegen.ecore/model/GenModel.genmodel"
```

Comment:
The plugin is missing a dependency to the uml2.codegen plugin (at least in my installation) adding it fixed the mwe2 build in eclipse

Reply:
That seems to be a problem with your installation. I do not have the problem, neither with Eclipse, nor with Maven. As the build server also does not complain, I think we are fine. The uml2 stuff comes in through the StoEx library. Do you maybe have an old version of the StoEx project in your Eclipse instance?